inst/extdata/gcam-china/CCS_provincial_data_add.csv

```diff
@@ -0,0 +1,19 @@
+# File: CCS_provincial_data_add.csv
+# Title: CO2 and Cost(1990USD) by province
```

Comment:
Is this per ton of C or CO2? Double check and use the correct one in the file name.

Reply:
get